public/js/render.js

```diff
@@ -14,10 +16,34 @@ var renderModule = function (error, eventsArray) {
     eventsSection.innerHTML = error;
   }
 
-  var filterEvents = eventsArray.filter(function (event) {
-    return event.en;
+  var filterEvents = [];
+
+  document.getElementById('arLang').addEventListener('click', function () {
+    type = 'Arabic';
+    getEventsModule({
+      date_from: new Date().toISOString().slice(0, 10)
+    }, renderModule);
+  });
+
+  document.getElementById('enLang').addEventListener('click', function () {
+    type = 'English';
+    getEventsModule({
+      date_from: new Date().toISOString().slice(0, 10)
+    }, renderModule);
   });
 
+  switch (type) {
```

Comment:
Neater for this just to be an `if / else` statement I think, since we're only planning on ever having two languages